darwin/importer/formats/labelbox_schemas.py

```diff
@@ -0,0 +1,78 @@
+bounding_box = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/bounding_box",
+    "description": "Schema of a Bounding Box",
+    "title": "Bounding Box",
+    "default": {"top": 1.2, "left": 2.5, "height": 10, "width": 20},
+    "examples": [{"top": 0, "left": 0, "height": 10, "width": 20}],
+    "type": "object",
+    "properties": {
+        "top": {"type": "number"},
+        "left": {"type": "number"},
+        "height": {"type": "number"},
+        "width": {"type": "number"},
+    },
+    "required": ["top", "left", "height", "width"],
+}
+
+point = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/point",
+    "description": "Schema of a Point",
+    "title": "Point",
+    "default": {"x": 1.2, "y": 2.5},
+    "examples": [{"x": 1.2, "y": 2.5}, {"x": 0, "y": 1}],
+    "type": "object",
+    "properties": {"x": {"type": "number"}, "y": {"type": "number"}},
+    "required": ["x", "y"],
+}
+
+polygon = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/polygon",
+    "description": "Schema of a Polygon",
+    "title": "Polygon",
+    "default": [{"x": 1.2, "y": 2.5}, {"x": 2.5, "y": 3.6}, {"x": 1.2, "y": 2.5}],
+    "examples": [[{"x": 1.2, "y": 2.5}, {"x": 2.5, "y": 3.6}, {"x": 1.2, "y": 2.5}], []],
+    "type": "array",
+    "items": point,
+}
+
+
+label_object = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Should it include `"$id"`?

Reply:
I am not sure all objects here should have `$id`. Reality is I subdivided this into so many objects because I the structure was quiet complex and big and this made it easier to understand. The only object I am actually reusing is `point`.

We can  have a talk about this if you want.